compiler/rustc_middle/src/hir/map/collector.rs

```diff
@@ -197,7 +142,15 @@ impl<'a, 'hir> NodeCollector<'a, 'hir> {
             }
         }
 
-        self.insert_entry(hir_id, entry, hash);
+        let data = &mut self.map[hir_id.owner];
+        let nodes = data.as_mut().unwrap();
```

Comment:
```suggestion
        let nodes = self.map[hir_id.owner].as_mut().unwrap();
```
Nit: `data` is used once and the name is too generic to be useful as a separate variable.